pkg/security/probe/dentry_resolver.go

```diff
@@ -305,6 +323,20 @@ func (dr *DentryResolver) Start() error {
 	return nil
 }
 
+// ErrTruncatedSegment is used to notify that a segment of the path was truncated because it was too long
+type ErrTruncatedSegment struct{}
```

Comment:
nit: Why not simply var ErrTruncatedSegment = errors.New("truncated_segment") ?

Reply:
In `custom_events.go` I use the error type de define the event type, imo type casting is cleaner that checking the error strings :)